<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>446</width>
    <height>198</height>
   </rect>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>Qt_Feng37_Bank</string>
  </property>
  <property name="toolTip">
   <string/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QComboBox" name="comboBox1">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>10</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>請選擇金融機構</string>
    </property>
    <item>
     <property name="text">
      <string>(013)國泰世華商業銀行</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>(017)兆豐國際商業銀行</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>(048)王道商業銀行</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>(103)臺灣新光商業銀行</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>(700)中華郵政股份有限公司</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>(812)台新國際商業銀行</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>(822)中國信託商業銀行</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>15</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>金融機構：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>存款金額：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>累積存款：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label1">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>80</y>
      <width>241</width>
      <height>16</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>累積存款</string>
    </property>
    <property name="text">
     <string>尚未開始計算</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button1">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>110</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>存檔之前記得先修改</string>
    </property>
    <property name="text">
     <string>修改</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button2">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>110</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>關閉程式之前不要忘記存檔</string>
    </property>
    <property name="text">
     <string>存檔</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="textbox1">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>50</y>
      <width>241</width>
      <height>21</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>請輸入整數數字</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="label5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>委任第五職等
簡任第十二職等
第12屆臺北市長
第23任總統
中央銀行鋒兄分行</string>
    </property>
    <property name="text">
     <string>彩蛋</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>446</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>語言</string>
    </property>
    <addaction name="actionChinese"/>
    <addaction name="actionEnglish"/>
    <addaction name="actionJapanese"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionChinese">
   <property name="text">
    <string>中文</string>
   </property>
  </action>
  <action name="actionEnglish">
   <property name="text">
    <string>English</string>
   </property>
  </action>
  <action name="actionJapanese">
   <property name="text">
    <string>日本語</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
